src/pages/api/projects/[id]/issues/[issueId]/nonprofit.ts

```diff
@@ -0,0 +1,46 @@
+import { Types } from "mongoose";
+import {
+  nonprofitGetIssue,
+  nonprofitUpdateIssue,
+} from "server/mongodb/actions/Issue";
+import APIWrapper from "server/utils/APIWrapper";
+import { NonprofitUpdateIssue, Role } from "src/utils/types";
+
+export default APIWrapper({
```

Comment:
For both of these endpoints, you should check to make sure that the nonprofit trying to access/modify a specific issue owns the associated project.  You can do that by calling nonprofitGetProject and seeing if you get the project.